etc/conf.d/nginx_lua.conf

```diff
@@ -1,5 +1,5 @@
 # You will probably want to use your own resolver here. It's necessary for Lua to function.
-resolver 8.8.8.8 8.8.4.4;
+resolver 127.0.0.11;
```

Comment:
Oh this is awesome! Could we add a short comment stating that this is the docker compose internal dns resolver?